### PR TITLE
Material Icon Sets, Damascus Steel Line Finish and Finally Damascus Steel Icon Sets

### PR DESCRIPTION
**Pull request by jude123412**
Added Class Files for Material Icon Set Creation along with Damascus Icon Sets Added Textures for all Damascus Icon Sets
Added Chemical Formula for Damascus Steel
Finished Damascus Steel Line, Now produces Ingots instead of Plates 
Changes to various code to make it more clean

### CODE REVIEW

File: src/main/java/gtb/api/unification/materials/GTBMaterialIconSetAddition.java
Comment (tekcay) [inline src/main/java/gtb/api/unification/materials/GTBMaterialIconSetAddition.java:1]: What I would suggest here to put everything IconSet related to the same class, `GTBMaterialIconType`. Move your `init()` method in there and rename it to something like `set()` or something. It avoids creating multiple classes

File: src/main/java/gtb/api/unification/materials/GTBMaterialSetFormula.java
Comment (tekcay) [inline src/main/java/gtb/api/unification/materials/GTBMaterialSetFormula.java:1]: Call me picky, but here I would name the class `GTBMaterialFormula` and the method to `set()`. That's my two cents :)

File: src/main/java/gtb/api/unification/materials/info/GTBMaterialIconSet.java
Comment (tekcay) [inline src/main/java/gtb/api/unification/materials/info/GTBMaterialIconSet.java:1]: `GTBMaterialIconType` has the same thing, so duplicate class, move it there :)

**Review by tekcay — CHANGES_REQUESTED** [review]
Everything's good. I'd just appreciate the little about IconSet stuff, but nothing too crazy, tell me what you think :)
Oh and finally a launch log without red lines !

**Comment by jude123412** [comment]
I agree and I'll get right on moving them 😉

**Comment by jude123412** [comment]
New commitment, hopefully its all ok :)

**Review by jude123412 — COMMENTED** [review]
Everything should be working properly now.

**Pull request closed without merging**